scripts/apps/search/directives/SearchParameters.ts

```diff
@@ -98,6 +123,21 @@ export function SearchParameters($location, asset, tags, metadata, common, desks
                     scope.fields.spike = 'exclude';
                 }
 
+                scope.dateFilters.forEach((dateFilter) => {
```

Comment:
Is the purpose of this block to set active filters from the URL? Can you add a comment before the block?